### PR TITLE
fix: insertion marker position when connection is resized (#7426)

### PR DESCRIPTION
**Pull request by clementcontet**
## The basics

- [x ] I [validated my changes](https://developers.google.com/blockly/guides/contribute/core#making_and_verifying_a_change)

### Resolves

Fixes https://github.com/google/blockly/issues/7426

### Proposed Changes

When positioning the insertion marker, take into account that its connection may have been resized.

### Reason for Changes

With these changes, when positioning the insertion marker, we ensure that the target connection will not move.

### Test Coverage

I tested in the playground with Zelos and Geras that insertion markers don't move target connections anymore, even if their connection have been resized.

### Documentation

N/A

### CODE REVIEW

File: core/insertion_marker_manager.ts [hunk 1]
```diff
@@ -611,12 +611,22 @@ export class InsertionMarkerManager {
...
-    insertionMarker.positionNearConnection(imConn, closest);
     // Connect() also renders the insertion marker.
     imConn.connect(closest);
 
+    const originalOffsetToTarget = {
+      x: closest.x - imConn.x,
+      y: closest.y - imConn.y,
+    };
+    const originalOffsetInBlock = imConn.getOffsetInBlock().clone();
+    const imConnConst = imConn;
```
Comment (rachel-fenichel) [inline core/insertion_marker_manager.ts:622]: nit: You don't need to assign this to another variable. You can just pass it directly.

**Review by rachel-fenichel — COMMENTED** [review]
Logic looks good. There's one nit to fix, and then I'll merge it.

File: core/insertion_marker_manager.ts (hunk 1 above)
Comment (clementcontet) [reply core/insertion_marker_manager.ts:622]: Hum, no.
These values are modified during the render.
That's why I get their "original" value in constants before the render.

Comment (rachel-fenichel) [reply core/insertion_marker_manager.ts:622]: Oh, I see.

**Review by rachel-fenichel — APPROVED** [review]

**Pull request merged**